functionary/core/models/function.py

```diff
@@ -28,6 +38,9 @@ class Function(models.Model):
     display_name = models.CharField(max_length=64, null=True)
     summary = models.CharField(max_length=128, null=True)
     description = models.TextField(null=True)
+    env_variables = models.JSONField(
```

Comment:
I vote to just call this `variables` here and in the package definition yaml too.